halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Pubsubs.java

```diff
@@ -27,10 +27,14 @@
 
 @Data
 @EqualsAndHashCode(callSuper = false)
-public class Pubsubs extends Node implements Cloneable {
+public class Pubsubs extends Node implements Cloneable, CanEnabled {
   private Boolean enabled;
   private GooglePubsub google = new GooglePubsub();
 
+  public boolean isEnabled() {
+    return enabled == null || !enabled.booleanValue();
```

Comment:
There's already a `getEnabled()` method that can be used for this. Also, the flag in this class is not used for anything, pubsubs is "enabled" only if every child pubsub type is enabled.

Reply:
The point is to make it `Activable` so it has to be `isEnabled`. We also need a `boolean` not a `Boolean`. And finally, if it's disabled, we don't need to look at children pubsub.